webpack/assets/javascripts/react_app/API.js

```diff
@@ -1,17 +1,31 @@
 import ServerActions from './actions/ServerActions';
 
 export default {
-  getStatisticsData(url) {
+  get(url) {
     $.ajaxPrefilter(function (options, originalOptions, jqXHR) {
       jqXHR.originalRequestOptions = originalOptions;
     });
-    $.getJSON(url)
+    return $.getJSON(url);
+  },
+  getStatisticsData(url) {
+    this.get(url)
       .success(
         (rawStatistics, textStatus, jqXHR) => {
           ServerActions.receivedStatistics(rawStatistics, textStatus, jqXHR);
         })
       .error((jqXHR, textStatus, errorThrown) => {
         ServerActions.statisticsRequestError(jqXHR, textStatus, errorThrown);
       });
+  },
+  getHostPowerData(url) {
```

Comment:
This feels to me like boilerplate, each call doing the same thing just calling a different ServerActions method... would it make sense to pass a parameter (for example, the URL) to a method in ServerActions that will just map the url to the relevant methods instead of having to add 10 lines for each new call?